### PR TITLE
add stylesheet

### PR DESCRIPTION
**Pull request by bsudekum**
test

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+    fontSize: 22,
+    fontWeight: 'normal'
+  },
+  h3: {
+    fontSize: 15,
+    fontWeight: 'normal'
+  },
+  h4: {
+    fontSize: 12,
+    fontWeight: 'normal'
```
Comment (rsudekum) [inline index.js:21]: fontWeight: 'normal'

File: index.js [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+  pad8y: {
+    paddingTop: 80,
+    paddingBottom: 80
+  },
+  pad8x: {
+    paddingLeft: 80,
+    paddingRight: 80
+  },
+  spaceBottom0: {
+    marginBottom: 5
```
Comment (rsudekum) [inline index.js:149]: And another. spaceBottom0 cool

File: index.js [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+    backgroundColor: '#3bb2d0'
+  },
+  fillCyan: {
+    backgroundColor: '#3bb2d0'
+  },
+  fillMidnight: {
+    backgroundColor: '#142736'
+  },
+  fillBlue: {
+    backgroundColor: '#4183c4'
```
Comment (rsudekum) [inline index.js:200]: fillBlue

**Comment by bsudekum** [comment]
Neat

**Comment by rsudekum** [comment]
Nice

**Comment by rsudekum** [comment]
foo

**Comment by rsudekum** [comment]
fff

**Comment by rsudekum** [comment]
dfsdf

**Comment by rsudekum** [comment]
ffoo

**Comment by bsudekum** [comment]
@bsudekum :+1:  looks good. :ship:

**Comment by bsudekum** [comment]
@bsudekum looks good. :+1: :ship:

**Comment by rsudekum** [comment]
bump

**Pull request merged**